Auto submodule update: cmake - Add clang format 11 (#119)

akleeman opened the pull request:
Automated PR by Jenkins. If CI has passed successfully, merge away!

**cmake**  600ec2a8  ->  62f063fe
-  62f063fe  :  Add clang format 11 (#119)
-  05a68330  :  Allow project wide default setting for exceptions and rtti (#118)
-  a1860df6  :  Add find module for starling-core (#116)
-  449f9b37  :  Add Module For Checking Atomic (#117)
-  63396d60  :  Sonarcloud submodule inclusion fix (#115)
-  6320bbfa  :  Upgrade Sonarcloud Functionality (#114)
-  93df75d4  :  Removed enabled features (#113)
-  4c16386e  :  Integrity unit tests framework (#111)
-  a4f91f7b  :  Change to using alias target
-  52ef1a33  :  Change Target Name
-  cd0ff55b  :  Change Optional Target
-  31604e72  :  clang-tidy: allow target to be disabled (#109)
-  f22a0f00  :  fix: use correct target types for compilable targets


This pull request was created by manual invocation.

Paakkit commented:
Did we get bunch of `log_error_sid(*sid, "TGD not applied for the signal");` errors somewhere?

nsirola commented:
Yes, when trying to use this repo in PFWP.

akleeman committed:
cmake - Add clang format 11
Triggered-By:	cmake	62f063fe21b19eaee6321407e02f9d9593f071c1
Upstream-PR:	http://github.com/swift-nav/cmake/pull/119
